src/pages/integrations/analytics/custom-events/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Custom events for Beacon analytics
+---
+
+# Creating custom events
+
+PWA Studio 12.5 contains an eventing framework with a series of default events that cover common scenarios.
+By creating and importing a custom extension, it is possible to add new events so that you may track specific user interactions beyond what is included in the default package.
+
+PWA Studio provides a [sample extension](https://github.com/magento/pwa-studio/tree/develop/packages/extensions/experience-platform-connector/) that shows how to add a custom event.
+
+In the sample extension, the `SEARCH_REPSONSE` event is being tracked. This event is fired when a search result is returned from the Adobe Experience Manager (AEM), with either results or a `EMPTY_RESULT` response.
```

Comment:
* I hope it's not spelled `SEARCH_REPSONSE` in the codebase 😬 
* the event is fired when Magento returns search results data, not AEM